hoodie-client/src/main/java/com/uber/hoodie/io/storage/HoodieWrapperFileSystem.java

```diff
@@ -121,12 +121,13 @@ private FSDataOutputStream wrapOutputStream(final Path path,
     if (fsDataOutputStream instanceof SizeAwareFSDataOutputStream) {
       return fsDataOutputStream;
     }
+    openStreams.remove(path.getName());
 
     SizeAwareFSDataOutputStream os =
         new SizeAwareFSDataOutputStream(fsDataOutputStream, new Runnable() {
           @Override
           public void run() {
-            openStreams.remove(path.getName());
+            // openStreams.remove(path.getName());
```

Comment:
Doing this helps to call fs.getBytesWritten(file) even after the stream is closed and gives back an exact number of uncompressed bytes written.

Reply:
stream is closed implies that the file (block i think) is fully written?

Reply:
Yes, and so the wrappedStream has the correct number of bytes to return..